samples/snippets/csharp/new-in-7/patternmatch.cs

```diff
@@ -9,13 +9,13 @@ namespace new_in_7
     #region 18_PercentileDie
     public struct PercentileDie
```

Comment:
Die or Dice? This distinction got me confused throughout the topic now that I went to see the comment.

Reply:
I updated the markdown to explain dice as the plural of die, and changed the code.